README.md

```diff
@@ -26,3 +26,164 @@ _If you get errors concerning the `FLASK_APP` environment not being set, try `$
 
 ### Database Schema
 ![image](https://user-images.githubusercontent.com/62635544/96819356-a6626380-13e0-11eb-8398-eef92ca100f3.png)
+
+### Endpoint Map
+
+### Applicants
+#### GET `/api/v1/applicants` 
+##### Response
+(all applicants)
+```
+{ 
+  data: [{
+​    "id": "1",
+​    "username": "Chipmunk",
+    "bio": "I'm the best one you could possibly hire",
+​    "updated_at": "Oct_20_etc_etc",
+​    "skills": ["javascript", "react"],
+​    "values": ["writing", "teamwork"]
+  }, {
+​    "id": "2",
+​    "username": "BeBe",
+    "bio": "I'm the best one you could possibly hire",
+​    "updated_at": "Oct_20_etc_etc",
+​    "skills": ["javascript", "react"],
+​    "values": ["paired programming", "magic"]
+  }, {
+​    "id": "3",
+​    "username": "Wingnut",
+    "bio": "I'm the best one you could possibly hire",
+​    "updated_at": "Oct_20_etc_etc",
+​    "skills": ["react"],
+​    "values": ["paired programming", "teamwork"]
+  }]
+}
+```
+#### GET `/api/v1/applicants/search-options` or `/api/v1/applicants/attributes`
```

Comment:
My vote is for`/api/v1/applicants/search-options` because we are referring to them as options throughout the codebase